platform/broadcom/docker-syncd-brcm-rpc/Dockerfile.j2

```diff
@@ -9,13 +9,14 @@ debs/{{ deb }}{{' '}}
 {%- endfor -%}
 debs/
 
-RUN dpkg -P syncd
-
 RUN dpkg_apt() { [ -f $1 ] && { dpkg -i $1 || apt-get -y install -f; } || return 1; } ; \
 {% for deb in docker_syncd_brcm_rpc_debs.split(' ') -%}
 dpkg_apt debs/{{ deb }}{{'; '}}
 {%- endfor %}
 
+# FIXME: Uncomment this line after libsaithrift is compatible with SAI v1.0
+# RUN dpkg -P syncd
+
```

Comment:
It's better to move this purge operation on previous place. We should remove old syncd before we install new one